fog/overseer/server/src/worker.rs

```diff
@@ -32,8 +32,8 @@ pub struct OverseerWorker {
     /// Join handle used to wait for the thread to terminate.
     join_handle: Option<JoinHandle<()>>,
 
-    /// Stop request trigger, used to signal the thread to stop.
-    stop_requested: Arc<AtomicBool>,
+    /// When true, overseer performs its monitoring logic.
+    is_enabled: Arc<AtomicBool>,
```

Comment:
Is there a need to store this here? If it comes from the outside (the caller of `OverseerWorker::new`) and is immediately passed to the thread, my thinking is no.

Reply:
I moved it out. However, for the new AtomicBool, stop_requested, I keep it in this struct because the stop method needs  to set it to true.